components/layout/search/filter/index.tsx

```diff
@@ -25,10 +26,14 @@ export default function FilterList({ list, title }: { list: ListItem[]; title?:
           </h3>
         ) : null}
         <ul className="hidden md:block">
-          <FilterItemList list={list} />
+          <Suspense fallback={null}>
```

Comment:
it's not documented yet, but can we assume that we need to wrap component with a `<Suspense>` when the component calls `useSearchParams()`?